### PR TITLE
feat(onyx-1279): prefetch tabs from new home view screen

### PR DESCRIPTION
**Pull request by nickskalkin**
This PR resolves [ONYX-1279]

### Description

Prefetch tabs just like how "old" home screen does it: https://github.com/artsy/eigen/blob/f179a02295479892cac6f0f379ae246711c5188d/src/app/Scenes/Home/Home.tsx#L145-L150

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#nochangelog

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[ONYX-1279]: https://artsyproduct.atlassian.net/browse/ONYX-1279?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by MounirDhahri — DISMISSED** [review]
👏👏

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ✅ **No changelog changes**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against eb9d4eb7b45fb282e5371a571e4f32030c7a9a04
</p>

File: src/app/Scenes/HomeView/HomeView.tsx [hunk 1]
```diff
@@ -41,9 +44,17 @@ export const HomeView: React.FC = () => {
   const savedArtworksCount = meData?.counts?.savedArtworks ?? 0
   useDismissSavedArtwork(savedArtworksCount > 0)
   useEnableProgressiveOnboarding()
+  const prefetchUrl = usePrefetch()
 
   const sections = extractNodes(data?.homeView.sectionsConnection)
 
+  useEffect(() => {
+    prefetchUrl<SearchQuery>("search", searchQueryDefaultVariables)
```
Comment (olerichter00) [inline src/app/Scenes/HomeView/HomeView.tsx:52]: 🤩

**Review by dblandin — APPROVED** [review]
This LGTM 👍🏼 

Similar to [this comment](https://github.com/artsy/eigen/pull/10769#pullrequestreview-2302849780) in another PR, this also seems like the sort of thing that doesn't necessarily need to live nested within the Home screen definition but we can look into that separately.

**Review by dariakoko — APPROVED** [review]
⭐

**Pull request merged**